src/main/views/features/sole-trader/correspondance-address-manual/capture-correspondance-address-manual.njk

```diff
@@ -8,7 +8,9 @@
 
 <main>
     <form action="correspondance-address-manual" method="POST">
-        <span class="govuk-caption-l">{{firstName}} {{lastName}}</span>
+        <span class="govuk-caption-l" style="max-width: 100%; overflow: hidden; text-overflow: ellipsis; white-space: normal;">
```

Comment:
Can we make it a common template and use it across the pages please?

Reply:
Sure.